xdsl/dialects/riscv_func.py

```diff
@@ -72,13 +72,13 @@ class CallOp(IRDLOperation):
     name = "riscv_func.call"
     args: VarOperand = var_operand_def(riscv.IntRegisterType)
```

Comment:
I don't see why we only change to `riscv.RISCVRegisterType` for results and not args here, am I missing something?

Reply:
No that's my bad